src/main/java/Timer.java

```diff
@@ -0,0 +1,35 @@
+import java.io.*;
+
+public class Timer {
+  private long startTime, endTime;
+  public long totalTime;
+
+  Timer() {
+    startTime = 0;
+    endTime = 0;
+    totalTime = 0;
+  }
+
+  public void start() {
+    startTime = System.nanoTime();
+  }
+
+  public void stop() {
+    endTime = System.nanoTime();
+    totalTime = endTime - startTime;
+  }
+
+  public void saveTime(String fileName) {
+    try {
+      File times = new File(fileName);
```

Comment:
Does this create a new file each time/overwrite the data or append it to the list?